hack/ci/lib/build_images.sh

```diff
@@ -39,6 +39,7 @@ build_images() {
     # Create an archive of docker images
     docker save \
         "${DOCKER_REPO}"/cert-manager-controller:"${DOCKER_TAG}" \
+        "${DOCKER_REPO}"/cert-manager-injectorcontroller:"${DOCKER_TAG}" \
```

Comment:
I think I prefer `cainjector` (i.e. the name of the actual go package) as the name here, unless there's some reason that won't work?